src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -156,3 +156,14 @@ def _complex_params(command_group):
         with self.argument_context(scope) as c:
             c.argument('server_name', options_list=['--name', '-s'])
             c.argument('kid', options_list=['--kid', '-k'], help='The Azure Key Vault key identifier of the server key. An example key identifier is "https://YourVaultName.vault.azure.net/keys/YourKeyName/01234567890123456789012345678901"')
+
+    for scope in ['mysql server key', 'postgres server key']:
+        with self.argument_context(scope) as c:
+            c.argument('server_name', options_list=['--name', '-s'])
```

Comment:
I will prefer using ['--server-name', '-s'] to keep consistent with existing commands.

Reply:
This section was not supposed to be there as its copy paste error and remove the same.